ios/ElectrodeApiImpl/APIImpls/Core/ENCoreDelegate.swift

```diff
@@ -51,6 +53,18 @@ import UIKit
         }
     }
 
+    @objc func didBlur() {
+        if viewController?.navigationController?.topViewController == viewController {
```

Comment:
are we checking if we have the same reference of ViewController? if so, i think should use === instead of ==

Reply:
updated